src/db/models.py

```diff
@@ -0,0 +1,94 @@
+import datetime as dt
+from sqlalchemy import Boolean, Column, ForeignKey, Integer, String, DateTime
+from sqlalchemy.orm import relationship
+
+from . import Base
+
+
+class User(Base):
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key=True, index=True)
+    email = Column(String, unique=True, index=True)
+    hashed_password = Column(String)
```

Comment:
Not sure if it will be needed cuz we use 3rd party auth, see supertokens for that ;)

Reply:
I see
I will leave it here for now to use it in develop and once we add supertokens it will be removed